src/ai_marketplace_monitor/facebook.py

```diff
@@ -568,23 +580,34 @@ def get_listing_details(
         return details, False
 
     def check_listing(
-        self: "FacebookMarketplace", item: Listing, item_config: FacebookItemConfig
+        self: "FacebookMarketplace",
+        item: Listing,
+        item_config: FacebookItemConfig,
+        description_available: bool = True,
     ) -> bool:
         # get antikeywords from both item_config or config
         antikeywords = item_config.antikeywords
-        if antikeywords and (
-            is_substring(antikeywords, item.title + " " + item.description, logger=self.logger)
-        ):
-            if self.logger:
-                self.logger.info(
-                    f"""{hilight("[Skip]", "fail")} Exclude {hilight(item.title)} due to {hilight("excluded keywords", "fail")}: {', '.join(antikeywords)}"""
-                )
-            return False
+        if antikeywords:
+            # Determine what text to search based on availability
+            search_text = item.title + (" " + item.description if description_available else "")
```

Comment:
The original code works. `item.description` will be an empty string if not retrieved.

Reply:
will revert this code section- I'm still trying to root-cause a different bug